server/portal/apps/system_monitor/urls.py

```diff
@@ -5,4 +5,5 @@
 app_name = 'system_monitor'
 urlpatterns = [
     path('', views.SysmonDataView.as_view(), name='system_monitor'),
+    path('<str:system_name>', views.SysmonDataView.as_view(), name='system_monitor'),
```

Comment:
Can we condense down to one path?

Reply:
I believe we could but for that we would have to have a slightly complex regex expression handling the routing. I would prefer the current way instead of regex since it is easier to understand at a quick glance what is happening. This approach would be similar to the approach in the DataFiles [urls.py](https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/datafiles/urls.py) file